README.md

```diff
@@ -56,209 +44,13 @@ If this is your first contact with the Orion Context Broker, it is highly recomm
-
-    yum install contextBroker-test
-
-or
-
-    rpm -i contextBroker-test-X.Y.Z-1.x86_64.rpm
-
-[Top](#top)
-
-### Upgrading from a previous version
-
-Upgrade procedure depends on whether the *upgrade path* (i.e. from the installed Orion version to the target one to upgrade) crosses a version number that requires:
-
-* Upgrading MongoDB version
-* Migrating the data stored in DB (due to a change in the DB data model).
-
-#### Upgrading MongoDB version
-
-You only need to pay attention to this if your upgrade path crosses 0.11.0 or 0.22.0. Otherwise, you can skip this section.
-
-* Orion versions previous to 0.11.0 recommend MongoDB 2.2
-* Orion version from 0.11.0 to 0.20.0 recommend MongoDB 2.4. Check [the 2.4 upgrade procedure in the oficial MongoDB documentation.](http://docs.mongodb.org/master/release-notes/2.4-upgrade/)
-* Orion version from 0.21.0 on recommend MongoDB 2.6 o 3.0. check [the 2.6 upgrade procedure](http://docs.mongodb.org/master/release-notes/2.6-upgrade/) or
-[the 3.0 upgrade procedure](http://docs.mongodb.org/master/release-notes/3.0-upgrade/) in the oficial MongoDB documentation.  
-
-#### Migrating the data stored in DB
-
-You only need to pay attention to this if your upgrade path crosses 0.14.1, 0.19.0 or 0.21.0. Otherwise, you can skip this section. You can also skip this section if your DB are not valuable (e.g. debug/testing environments) and you can flush your DB before upgrading.
-
-* [Upgrading to 0.14.1 and beyond from a pre-0.14.1 version](doc/manuals/admin/upgrading_crossing_0-14-1.md)
-* [Upgrading to 0.19.0 and beyond from a pre-0.19.0 version](doc/manuals/admin/upgrading_crossing_0-19-0.md)
-* [Upgrading to 0.21.0 and beyond from a pre-0.21.0 version](doc/manuals/admin/upgrading_crossing_0-21-0.md)
-
-If your upgrade cover several segments (e.g. you are using 0.13.0 and
-want to upgrade to 0.19.0, so both "upgrading to 0.14.1 and beyond from
-a pre-0.14.1 version" and "upgrading to 0.19.0 and beyond from a
-pre-0.19.0 version" applies to the case) you need to execute the
-segments in sequence (the common part are done only one time, i.e. stop
-CB, remove package, install package, start CB). In the case of doubt,
-please [ask using StackOverflow](http://stackoverflow.com/questions/ask)
-(remember to include the "fiware-orion" tag in your questions).
-
-#### Standard upgrade 
-
-If you are using yum, then you can upgrade doing (as root):
-
-` yum install contextBroker`
-
-Sometimes the above commands fails due to yum cache. In that case, run
-`yum clean all` and try again.
-
-If you are upgrading using the RPM file, then first download the new package from the [FIWARE Files area](https://forge.fiware.org/frs/?group_id=7). Look for the "DATA-OrionContextBroker" entry.
-
-Then upgrade the package using the rpm command (as root):
-
-    rpm -U contextBroker-X.Y.Z-1.x86_64.rpm
+Orion Context broker is described in detail in [the following document](doc/manuals/admin/install.md).
 
```

Comment:
What about

Build and Install documentation for Orion Context Broker can be found at ...